packages/patternfly-4/react-core/src/components/List/List.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/List/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /** Anything that can be rendered inside of the list */
+  children: PropTypes.node,
```

Comment:
Do we want anything here, or Do we want it to be of type ListItem?  The permitted content for a list is zero or more list item nodes.

Reply:
@tlabaj I'm not familiar with the difference. Is there a benefit of using type ListItem vs leaving it as is?

Reply:
Would it be something like: `children: React.PropTypes.oneOf([List])`

Reply:
Children is hard to type check, and also can be a little annoying for consumers. Children can be an array, null, false, string, number, function. The only time it is usually valuable to use PropTypes with it (other than node) is when you expect a function. 

It is commonplace to do something like

```
const SomeWrappedListItem = (props) => <ListItem {...props} someDefaultProp />

<List>
  <SomeWrappedListItem />
</List>

```
If the lib tried to force a type as a child it can create a lot of noise in the console.

Reply:
Thanks @dmiller9911, I think given that this is a simple list, it'll be better to keep as is to avoid any additional complexity that's not needed.

Reply:
Thanks @dmiller9911, I think given that this is a simple list, it'll be better to keep as is to avoid any additional complexity if that's fine with you as well @tlabaj .